scripts/build_renewable_profiles.py

```diff
@@ -238,8 +238,15 @@
 
     cutout = atlite.Cutout(paths["cutout"])
     regions = gpd.read_file(paths.regions).set_index("name").rename_axis("bus")
+    regions_crs=regions.crs
     # TODO: Drop NAN maybe somewhere else? NaN lead to error otherwise
     regions = regions.dropna(axis="index", subset=["geometry"])
+    if snakemake.config['alternative_clustering']:
```

Comment:
In https://github.com/pypsa-meets-africa/pypsa-africa/pull/134/files#r744323030 I was referring to this if